packages/cli/src/index.ts

```diff
@@ -410,6 +412,36 @@ yargs
       })(argv);
     }
   )
+  .command(
+    'preview',
+    'Preview Redocly project',
+    (yargs) =>
+      yargs.options({
+        product: {
+          type: 'string',
+          choices: ['redoc', 'revel', 'reef', 'realm', 'redoc-revel', 'redoc-reef', 'revel-reef'],
+        },
+        plan: {
+          type: 'string',
+          choices: PRODUCT_PLANS,
+          default: 'enterprise',
+          description: '',
+        },
+        port: {
```

Comment:
I don't see the `port` to be used anywhere. Is it intentional? 

Do we also need the `host` option similar to [this](https://github.com/Redocly/redocly-cli/blob/main/packages/cli/src/index.ts#L431)?
See [the issue](https://github.com/Redocly/redocly-cli/issues/474) for the `preview-docs` command.

Reply:
Nice catch. Added use of the port argument. 

The `0.0.0.0` address is not allowed in the project preview, so there's no need for `host` option.